src/expression.js

```diff
@@ -112,7 +120,13 @@ var Lookup = function(key, root) {
 	this.rootExpr = root;
 };
 Lookup.prototype.value = function(scope, helperOptions){
-	var result = lookupValueOrHelper(this.key, scope, helperOptions);
+	var result = {};
+
+	if (this.rootExpr instanceof Call) {
```

Comment:
My thought is this would be 

```
if(this.rootExpr) { 
  ...
}
```

If there is a rootExpt all, this should be getting the value from that expression.  It's semi possible to do `{{"foo".length}}`